docker-compose.yml

```diff
@@ -123,8 +127,6 @@ services:
         volumes:
             - ./db-manager/dummy_csv:/dummy_csv
             - csv-data:/csv-data
-        ports:
-          - 5432:5432
```

Comment:
Not sure why the postgres port was used for db-manager, probably by mistake. Was conflicting with my local postgres db.